app/assets/stylesheets/common/_markus.scss

```diff
@@ -1003,11 +1003,12 @@ nav {
     display: none;
     left: -1px;
     margin: 0;
-    min-width: 16ch;
     padding: 0;
     position: absolute;
     text-align: left;
-    top: 1.95em;
+    top: 2.2em;
+    width: 101%;
```

Comment:
See comment below about `width: 101%`. Also the word-wrap rule you added was good! Should solve the issue you described.

Reply:
1.95em slightly overlays the display name box

Reply:
Declaring with a variable for 200px works for the normal view, however once switched into a role, the dropdown becomes much smaller than the box, even if I switch to using min-width.